src/validation.cpp

```diff
@@ -1451,6 +1451,36 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
     return true;
 }
 
+bool UndoReadFromDisk(CBlockUndo& blockundo, const CBlockIndex* pindex)
```

Comment:
Could split move only to a different commit.

Reply:
it will get squashed anyway no?